react/index.js

```diff
@@ -22,6 +22,8 @@ export { default as Secondary } from './Secondary/Secondary';
 export { default as Strong } from './Strong/Strong';
 export { default as Text } from './Text/Text';
 export { default as TextLink } from './TextLink/TextLink';
+export { default as BulletPoint } from './BulletPoint/BulletPoint';
```

Comment:
What scenario should we need this exposed? If it's not required at the moment, i would limit increasing the api footprint to just the list for now.